syntax = "proto3";

package tomchain;

service TcPeerConsensus {
    rpc RelayVote(RelayVoteRequest)
        returns (RelayVoteResponse); 
    rpc RelayBlock(RelayBlockRequest)
        returns (RelayBlockResponse);
}

message RelayVoteRequest {
    uint32 id = 1;
    repeated bytes votes = 2;
}

message RelayVoteResponse {
    uint32 status = 1; 
}

message RelayBlockRequest {
    uint32 id = 1;
    repeated bytes block = 2; 
}

message RelayBlockResponse {
    uint32 status = 1; 
}
